design/how-ironic-works.md

```diff
@@ -146,6 +147,66 @@ Between ironic and ironic-inspector:
 * Each service must be able to reach the API endpoint for
   the the other service.
 
+## What is ironic-inspector?
+Ironic-inspector is an auxiliary service that provides separate API to
+inspect and register hardware properties for a bare metal node (node
+introspection process).
+This process may be triggered during discovery and allow automatic node
+registration in ironic.
+The process of "hardware introspection" allows also to get the hardware
+parameters from a bare metal node and ready it for scheduling.
+Power management credentials should be already set to allow node
+introspection.
+
+To start the introspection process using ironic-inspector API
+
+    POST /v1/introspection/node-id
```

Comment:
Does the body need any information, or is the URL enough to start the process? I assume the node needs to be registered in ironic already so they inspector will fetch anything it needs from there?

Reply:
I expanded the description trying to give more info on the status of the node in respect to ironic and added information on the body in request/response.

Reply:
Thanks!